controllers/docluster_controller.go

```diff
@@ -133,7 +133,13 @@ func (r *DOClusterReconciler) reconcile(ctx context.Context, clusterScope *scope
 	apiServerLoadbalancer.ApplyDefault()
 
 	apiServerLoadbalancerRef := clusterScope.APIServerLoadbalancersRef()
-	loadbalancer, err := networkingsvc.GetLoadBalancer(apiServerLoadbalancerRef.ResourceID)
+	lbUUID := apiServerLoadbalancerRef.ResourceID
+
+	if apiServerLoadbalancer.ResourceId != "" {
```

Comment:
Yes, I left this intentionally so that we don't break older api objects. Once we can be sure that we're no longer relying on the status, we can get rid of it.